src/style.css

```diff
@@ -0,0 +1,92 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  --height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  flex-basis: var(--height);
+
+  height: var(--height);
+
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+.block:nth-child(1) {
+  background-color: rgb(255, 0, 0);
+}
+
+.block:nth-child(2) {
+  background-color: rgb(204, 0, 0);
+}
+
+.block:nth-child(3) {
+  background-color: rgb(153, 0, 0);
+}
+
+.block:nth-child(4) {
+  background-color: rgb(102, 0, 0);
+}
+
+.block:nth-child(5) {
+  background-color: rgb(51, 0, 0);
+}
+
+.block:nth-child(6) {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 600px) {
+
+  .block:nth-child(1) {
+    order: 1;
+  }
+
+  .block:nth-child(2) {
+    order: 2;
+  }
```

Comment:
these are redundant, these blocks will be in their place by default, because they have `order: 0` - default one
```suggestion
```

Reply:
Thanks for the hint!